tmail-backend/tmail-third-party/openpaas/pom.xml

```diff
@@ -37,6 +37,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>apache-james-backends-rabbitmq</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>jmap-extensions</artifactId>
```

Comment:
I think we did fight hard not to get this dependency here.

Why do we get it?

How can we remove it?

Reply:
Not handled

Reply:
I'm aware of it. I know it can be handled, but it takes some refactoring on the jmap-extensions side. I need to get the CalDAV stuff to work to understand what changes need to be made there.

Reply:
(I'm picky but it is still not handled...)

Reply:
> jmap-extensions

I'll review this work only when the dependency will no longer be in the OpenPaaS module.